pybotvac/robot.py

```diff
@@ -46,7 +43,7 @@ def __init__(self, serial, secret, traits, vendor=Neato, name='',
         self._headers = {'Accept': 'application/vnd.neato.nucleo.v1'}
 
         if self.service_version not in SUPPORTED_SERVICES:
-            raise UnsupportedDevice("Version " + self.service_version + " of service houseCleaning is not known")
+            raise NeatoRobotException("Version " + self.service_version + " of service houseCleaning is not known")
```

Comment:
I would stay with unsupported device here, but let it extend the NeatoRobotException. What do you think?

Reply:
Ok lets do that.  I will do a slight rename so it matches the rest of the exceptions.